chart/templates/_helpers.tpl

```diff
@@ -50,6 +50,46 @@ app.kubernetes.io/name: {{ include "microengine-webhooks-py.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
+{{/*
+Nginx labels
+*/}}
+{{- define "microengine-webhooks-py.nginx.labels" -}}
+helm.sh/chart: {{ include "microengine-webhooks-py.chart" . }}
+{{ include "microengine-webhooks-py.nginx.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Nginx selector labels
+*/}}
+{{- define "microengine-webhooks-py.nginx.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "microengine-webhooks-py.name" . }}-nginx
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Worker labels
+*/}}
+{{- define "microengine-webhooks-py.worker.labels" -}}
+helm.sh/chart: {{ include "microengine-webhooks-py.chart" . }}
+{{ include "microengine-webhooks-py.worker.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Worker selector labels
+*/}}
+{{- define "microengine-webhooks-py.worker.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "microengine-webhooks-py.name" . }}-worker
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
```

Comment:
On Helm charts, it feels specially hard to figure out the correctness of the changes by glancing at the code.

Would be nice to have a diff of the `make template` somehow on the PR.

If I remember, will try to automate this diffing thing, at least up to a text to be copy-pasted.